pom.xml

```diff
@@ -81,6 +81,22 @@
     <!-- Libraries -->
     <dependencies>
         <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
```

Comment:
can you remove the jackson dependencies as you do not appear to use them in your code change below?